### DISCUSSION TITLE
👋 Questions & Community Support

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#2686] [Archive]
Join our [Discord Server](https://directus.chat/) to ask questions, get product updates or informally chat with other community members!

**Comment by ammaraga** [comment]
Hi!
I might be missing something, but is there a way to refresh a Directus collection to reflect changes in underlaying Database? In my case, I have a table which got extended by few fields, but matching Directus collection doesn't reflect those fields (or relationships). It would be nice to have a "refresh" button on collection level.
I'm currently on 9.12.1, using Azure SQL Database Server
